### PR TITLE
Add Tables

### PR DESCRIPTION
**Pull request by AHP15**
In this PR, I added the following:

 - A table named owners
 - A table named species
 - Modify animals table
 - Insert  data into the owners table
 - Insert  data into the species table
 - Modify inserted animals so it includes the species_id value
 - Modify inserted animals to include owner information (owner_id)
 - Write queries (using JOIN) to answer the following questions:
  
What animals belong to Melody Pond?

List of all animals that are pokemon (their type is Pokemon).

List all owners and their animals, remember to include those that don't own any animals

How many animals are there per species?

List all Digimon owned by Jennifer Orwell.

List all animals owned by Dean Winchester that haven't tried to escape.

Who owns the most animals?

# Cheers 👍

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -9,3 +9,38 @@ CREATE TABLE animals (
...
+
+/* owners */
+CREATE SEQUENCE owner_id_seq;
+
+create table owners (
+	id int primary key default nextval('owner_id_seq'),
+	full_name VARCHAR(255),
+	age int
+);
+
```
Comment (1GORDON) [inline schema.sql:21]: Great job @Abdessittir 🤝

- [ ] As stated in the project requirements, kindly add a new table with the name species with the following columns.
species

File: data.sql [hunk 2]
```diff
@@ -13,3 +13,61 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (10, 'Blossom', '1998-09-13', 3, true, 17);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (11, 'Ditto', '2022-05-14', 4, true, 22);
 
+/* Insert the following data into the owners table*/
+insert into owners (full_name, age)
+values ('Sam Smith', 34);
+
+insert into owners (full_name, age)
+values ('Jennifer Orwell', 19);
+
```
Comment (1GORDON) [inline data.sql:22]: Great job @Abdessittir 🤝

- [ ] You may also add this data to the `owners` table. You seem to have missed it.
bob

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Abdessittir ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @Abdessittir 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)

## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅
 - [ ]  Correct GitFlow. ✅
 - [ ]  Good commit history. ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**